src/swagger-2.ts

```diff
@@ -181,7 +198,7 @@ function parse(spec: Swagger2, namespace: string) {
 
   output.push('}'); // Close namespace
 
-  return prettier.format(output.join('\n'), { parser: 'typescript' });
+  return prettier.format(output.join('\n'), { parser: 'typescript', singleQuote: true });
```

Comment:
Just a note: I personally like singleQuote, but since we’re open-sourcing it I didn’t know the best thing to decide on. Maybe we could add options one day to allow customization of this, but this is fine for now (we can always change it if someone asks!)